internal/resources/fabric/service_token/resource.go

```diff
@@ -115,7 +131,7 @@ func resourceDelete(ctx context.Context, d *schema.ResourceData, meta interface{
 	return diags
 }
 
-func waitForStability(uuid string, meta interface{}, d *schema.ResourceData, ctx context.Context, tieout time.Duration) error {
+func waitForStability(uuid string, meta interface{}, d *schema.ResourceData, ctx context.Context, timeout time.Duration) (*fabricv4.ServiceToken, error) {
 	log.Printf("Waiting for service token to be created, uuid %s", uuid)
 	stateConf := &retry.StateChangeConf{
 		Target: []string{
```

Comment:
The line right below this says `string(fabricv4.SERVICETOKENSTATE_INACTIVE)` is the expected target. Shouldn't it be `string(fabricv4.SERVICETOKENSTATE_ACTIVE)`?

Reply:
The service token will be in ACTIVE status once it has been used to create a connection and that connection is active. When the token is ready to be used, it will be in INACTIVE status.